addons/editor-theme3/theme3.js

```diff
@@ -370,6 +371,13 @@ export default async function ({ addon, console, msg }) {
     }
   };
 
+  const oldInsertionMarkerCreateMarkerBlock = Blockly.InsertionMarkerManager.prototype.createMarkerBlock_;
+  Blockly.InsertionMarkerManager.prototype.createMarkerBlock_ = function (originalBlock) {
```

Comment:
This doesn't check if the setting is actually enabled.

Reply:
Also ideally this should support dynamic settings and disable.

Reply:
Whoops, I accidentally removed the check when I moved it to a new function. I'll fix it tomorrow